### DISCUSSION TITLE
new version kernel module support pcie p2p for 30s or 40s Geforce cards

### DISCUSSION THREAD

**Discussion by yongmei04** [NVIDIA/open-gpu-kernel-modules#629] [General]
My friends told me that pcie p2p will be enable against 550 version after enabling resize bar for bar1 by bios

**Comment by yongmei04** [comment]
does it relates to egm p2p support?
anyway, it's great feature for geforce guys.

**Reply by ALEX5402** [reply]
Ya Nvidia doing great these days,
